scripts/bnsd/genesis_app_state.json

```diff
@@ -89,6 +89,25 @@
       }
     ]
   },
+  "update_validators": {
+    "addresses": ["cond:gov/rule/0000000000000002"]
+  },
+  "escrow": [
+    {
+      "//name": "Guarantee Fund",
+      "amount": [
+        {
+          "ticker": "CASH",
+          "whole": 100000000000000
+        }
+      ],
+      "arbiter": "cond:gov/rule/0000000000000002",
+      "//destination-name": "Reward Fund",
+      "destination": "cond:gov/rule/0000000000000002",
+      "source": "0000000000000000000000000000000000000000",
+      "timeout": "2999-12-31T00:00:00Z"
+    }
+  ],
```

Comment:
Is it possible to sync this genesis with IOV-Core in a way that we can copy the entire `./bnsd/` folder from IOV-Core to the ponferrada repo? This has been the process so far and it saves a lot of time.

Reply:
You mean close this PR and open an equivalent one in IOV-Core instead?

Reply:
I don't mind in which way the sync is achieved. I would just be very happy if at the end of this process both repos have BNS governance tests that pass using the same genesis file.

Reply:
Oh, well all I did in this commit was copy and paste the genesis file over: https://github.com/iov-one/ponferrada/pull/499/commits/27eed130e500322aca2dca0f90797fdad61e35d4

I think it's easiest if we keep IOV-Core as the single source of truth though, and just periodically copy and paste the whole directory.

Reply:
https://github.com/iov-one/iov-core/pull/1178

Reply:
In this case we are on the same page already, perfect